core/shared/src/main/scala/chartreuse/Coordinate.scala

```diff
@@ -16,11 +16,14 @@
 
 package chartreuse
 
+import doodle.algebra.*
 import doodle.core.Point
 
 opaque type Coordinate = Point
 opaque type ScreenCoordinate <: Coordinate = Point
 opaque type DataCoordinate <: Coordinate = Point
+type TicksSequence = Seq[(ScreenCoordinate, DataCoordinate)]
+type PlotAlg = Layout & Path & Style & Shape & Text & doodle.algebra.Transform
```

Comment:
I moved these types out of `Plot` to make them accessible from `Axes`. But I'm not sure they are supposed to be in `Coordinate.scala`. Where should we put them?

Reply:
Honestly, I'm not sure either. `TicksSequence` could probably be in the `Axes` class, as I don't think it is needed outside of there. `PlotAlg` could probably be in a companion object for `Plot` as it's associated with `Plot`.